velox/functions/prestosql/window/WindowFunctionsRegistration.cpp

```diff
@@ -14,15 +14,20 @@
  * limitations under the License.
  */
 #include "velox/functions/prestosql/window/WindowFunctionsRegistration.h"
+#include "velox/exec/WindowFunction.h"
```

Comment:
Why is this include needed ? Please remove it if not.

Reply:
Sorry I missed this when reviewing the new commit, removed it now